Documentation/Build/DownloadExampleHighlighted.png.cid

```diff
@@ -0,0 +1 @@
+bafkreihapw3rzwmabx6k4kdwqi4feovykoxceoqfhksxbowd6odj5qdvpu
```

Comment:
Could you please add documentation on how these CIDs should be generated?

Reply:
I think that once a file is added to IPFS, it gets a CID.

Reply:
It would be nice to have documentation walkthrough for new/inexperienced users, particularly as most in the ITK community will be new to IPFS. Documentation could be copied or expanded from https://github.com/spatial-image/multiscale-spatial-image#development .

Reply:
Please see https://github.com/InsightSoftwareConsortium/ITKSphinxExamples/pull/398/commits/d0a4c9c1e7c1c86ff9c85c73d99e36cdf1115bd0